api-report/map.api.md

```diff
@@ -110,9 +110,9 @@ export interface ISharedDirectory extends ISharedObject<ISharedDirectoryEvents &
 // @public
 export interface ISharedDirectoryEvents extends ISharedObjectEvents {
     // (undocumented)
-    (event: "valueChanged", listener: (changed: IDirectoryValueChanged, local: boolean, target: IEventThisPlaceHolder) => void): any;
+    (event: "valueChanged", listener: (changed: IDirectoryValueChanged, local: boolean, op: ISequencedDocumentMessage | null, target: IEventThisPlaceHolder) => void): any;
```

Comment:
You have some issues with merge. Can you please double check? Many changes in .api.md files just do not make sense and they seem to come from you having old content, i.e. overwriting changes in main when merging main changes.